src/query/storages/fuse/src/table_functions/infer_schema/infer_schema_table.rs

```diff
@@ -117,12 +116,7 @@ impl Table for InferSchemaTable {
     }
 
     fn table_args(&self) -> Option<TableArgs> {
-        let mut args = HashMap::new();
```

Comment:
@Xuanwo here, all table function use this for serde and reconstruct (e.g. for distributed exexutition)

Reply:
Thanks, I got it.